md/taxonomy.md

```diff
@@ -292,3 +284,11 @@
   * [Use Groovy in Bonita](groovy-in-bonita.md)
   * [Create a responsive web application with Community edition](responsiveness-with-community-edition.md)
   * [Enable Adaptive Case Management with Bonita](use-bonita-acm.md)
+  * [Create an event handler](event-handlers.md)
+* [Contributing](_collaboration.md)
+  * [Build Bonita Community edition from source files](building-community-edition-from-source.md)
+  * [Bonita Engine architecture](_engine-architecture.md)
+    * [Bonita Engine architecture overview](engine-architecture-overview.md)
```

Comment:
I find it weird that engine-architecture-overview.md is at the same level than engine-architecture.md